modules/system/console/MixWatch.php

```diff
@@ -122,4 +133,21 @@ public function handleSignal(int $signal): void
             exit(0);
         }
     }
+
+    /**
+     * Handles the provided Windows process singal.
+     */
+    public function handleWindowsSignal(int $event): void
+    {
+        if ($event === PHP_WINDOWS_EVENT_CTRL_C || $event === PHP_WINDOWS_EVENT_CTRL_BREAK) {
+            // Cleanup
```

Comment:
The conflicting signal handling code merged with @bennothommo's commit https://github.com/wintercms/winter/commit/452bb4222351b1a6d3cc0288a30804ebeb37234d